src/python/pants/engine/internals/native.py

```diff
@@ -136,6 +136,9 @@ def init_rust_logging(
             level, log_show_rust_3rdparty, use_color, show_target, log_levels_as_ints
         )
 
+    def set_per_run_log_path(self, path: Optional[str]) -> None:
```

Comment:
can it really be None ?

Reply:
Yes calling this method with a `None` argument tells the rust logging code to avoid writing logs to any per-run file.

Reply:
Will document.